common/saber_base.h

```diff
@@ -74,7 +74,7 @@ class SaberBase {
   // 1.0 = kDefaultVolume
   // This is really just for sound fonts.
   virtual void SetHumVolume(float volume) {}
-  virtual void StartSwing() {}
+  virtual void StartSwing(float swing_acceleration_) {}
```

Comment:
underscore at end is only for variables in objects

Reply:
> In general, how is the acceleration thing working for you?
> Does it detect the difference between a slash and a swing?

It is working pretty well, yes.  I need to test with a different font but with the font I've been using so far, it's been consistent.

Reply:
further testing shows it working very consistently.